### PR TITLE
Renaming master branch references to main branch

### PR DESCRIPTION
**Pull request by razo7**
- Renaming `master` branch references to `main` branch (Why main was chosen? see [here](https://github.com/github/renaming#why-main)).
- Adding apostrophe

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ available, but also supports other mechanisms.
 
 Apart from the medik8s team being actively involved with the [Machine
 Healthcheck
-Controller](https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/machinehealthcheck_controller.go),
+Controller](https://github.com/kubernetes-sigs/cluster-api/blob/main/controllers/machinehealthcheck_controller.go),
```
Comment (razo7) [inline faq.md]: Looking at it again, there is no `machinehealthcheck_controller.go` under `https://github.com/kubernetes-sigs/cluster-api/blob/main/controllers`. Can it be outdated and there is no such file anymore?

File: _config.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ last_edit_time_format: "%b %e %Y at %I:%M %p" # uses ruby's time format: https:/
 gh_edit_link: true # show or hide edit this page link
 gh_edit_link_text: "Edit this page on GitHub"
 gh_edit_repository: "https://github.com/medik8s/docs" # the github URL for your repo
-gh_edit_branch: "master" # the branch that your docs is served from
+gh_edit_branch: "main" # the branch that your docs is served from
```
Comment (slintes) [inline _config.yml]: this should be `gh-pages` IMHO

File: faq.md (hunk 1 above)
Comment (slintes) [reply faq.md]: https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/controllers/machinehealthcheck/machinehealthcheck_controller.go

**Review by slintes — CHANGES_REQUESTED** [review]

**Review by slintes — APPROVED** [review]

**Comment by slintes** [comment]
merge conflict 🤷🏼‍♂️

**Pull request merged**